### PR TITLE
Custom Javascript URL list

### PR DESCRIPTION
**Pull request by sbrunner**
With vite in dev mode we should be able to inject 2 different Javascript...

### CODE REVIEW

File: contribs/gmf/src/controllers/bootstrap.js [hunk 1]
```diff
@@ -58,40 +79,51 @@ function bootstrap(module) {
...
+      if ('gmfCustomJavascriptUrl' in dynamic.constants) {
+        const gmfCustomJavascriptUrl = dynamic.constants.gmfCustomJavascriptUrl;
+        if (Array.isArray(gmfCustomJavascriptUrl)) {
+          gmfCustomJavascriptUrl.forEach((url) => {
+            addScript(url);
+          });
+        } else {
+          addScript(gmfCustomJavascriptUrl);
+        }
+      }
```
Comment (ger-benjamin) [inline contribs/gmf/src/controllers/bootstrap.js:103]: Eventually a little bit more concise:

```suggestion
      if ('gmfCustomJavascriptUrl' in dynamic.constants) {
        const gmfCustomJavascriptUrl = Array.isArray(dynamic.constants.gmfCustomJavascriptUrl) ? dynamic.constants.gmfCustomJavascriptUrl : [dynamic.constants.gmfCustomJavascriptUrl];
        gmfCustomJavascriptUrl.forEach((url) => {
          addScript(url);
        });
      }
```

File: contribs/gmf/src/controllers/bootstrap.js [hunk 2]
```diff
@@ -58,40 +79,51 @@ function bootstrap(module) {
...
+      if ('gmfCustomStylesheetUrl' in dynamic.constants) {
+        const gmfCustomStylesheetUrl = dynamic.constants.gmfCustomStylesheetUrl;
+        if (Array.isArray(gmfCustomStylesheetUrl)) {
+          gmfCustomStylesheetUrl.forEach((url) => {
+            addStylesheet(url);
+          });
+        } else {
+          addStylesheet(gmfCustomStylesheetUrl);
+        }
+      }
```
Comment (ger-benjamin) [inline contribs/gmf/src/controllers/bootstrap.js:113]:
```suggestion
      if ('gmfCustomStylesheetUrl' in dynamic.constants) {
        const gmfCustomStylesheetUrl = Array.isArray(dynamic.constants.gmfCustomStylesheetUrl) ? dynamic.constants.gmfCustomStylesheetUrl : [dynamic.constants.gmfCustomStylesheetUrl];
        gmfCustomStylesheetUrl.forEach((url) => {
          addStylesheet(url);
        });
      }
```

File: contribs/gmf/src/controllers/bootstrap.js [hunk 3]
```diff
@@ -58,40 +79,51 @@ function bootstrap(module) {
...
-      module.constant(name, dynamic.constants[name]);
-    }
+      for (const name in dynamic.constants) {
+        module.constant(name, dynamic.constants[name]);
+      }
 
-    setupI18n();
-    angular.bootstrap(document, [`App${interface_}`]);
-  });
+      setupI18n();
```
Comment (ger-benjamin) [inline contribs/gmf/src/controllers/bootstrap.js:121]:
```suggestion
    setupI18n();
```

File: contribs/gmf/src/controllers/bootstrap.js [hunk 4]
```diff
@@ -58,40 +79,51 @@ function bootstrap(module) {
...
+      angular.bootstrap(document, [`App${interface_}`]);
+    })
+    .catch((error) => {
+      window.alert(`Failed to get the dynamic: ${error}`);
```
Comment (ger-benjamin) [inline contribs/gmf/src/controllers/bootstrap.js:125]: With your `fail` function, we probably don't need this catch anymore. Or at least the message is now wrong

Comment (sbrunner) [reply contribs/gmf/src/controllers/bootstrap.js:125]: Eslint didn't agree, I add it to remove a working that covers all my code...

File: contribs/gmf/src/controllers/bootstrap.js (hunk 3 above)
Comment (sbrunner) [reply contribs/gmf/src/controllers/bootstrap.js:121]: I don't see that ...

File: contribs/gmf/src/controllers/bootstrap.js (hunk 4 above)
Comment (ger-benjamin) [reply contribs/gmf/src/controllers/bootstrap.js:121]: okay...

**Review by ger-benjamin — APPROVED** [review]

**Review by llienher — APPROVED** [review]

**Pull request merged**